wagtail/admin/templates/wagtailadmin/userbar/item_admin.html

```diff
@@ -1,8 +1,11 @@
 {% extends "wagtailadmin/userbar/item_base.html" %}
-{% load i18n  %}
+{% load i18n wagtailadmin_tags %}
 
 {% block item_content %}
-    <div class="wagtail-action wagtail-icon wagtail-icon-wagtail">
-        <a href="{% url 'wagtailadmin_home' %}" target="_parent" class="">{% trans 'Go to Wagtail admin' %}</a>
+    <div class="wagtail-action">
+        <a href="{% url 'wagtailadmin_home' %}" target="_parent" class="">
```

Comment:
Is this empty `class` attribute needed?

```suggestion
        <a href="{% url 'wagtailadmin_home' %}" target="_parent">
```